view/index.html

```diff
@@ -13,7 +13,7 @@
     <script src="./js/services/d3.v4.js"></script>
     <link rel="stylesheet" href="./css/main.css">
 </head>
-<body background="./view/assest/Background/MainBG.png">
+<body background="../assets/Background/MainBG.png">
```

Comment:
@irshadjsr21 Have you tried to run the code or have changed just manually as it is also not the correct path for the image and moreover, the background image mentioned was for the previous UI, it has been changed now, that is,  we don't require it for the timing. Can you please remove the `background` styling from the `body tag`.

Reply:
@muskankhedia Yes, the background works with this path in my system. In fact, all the other images in the app use a similar path.

**Example**
https://github.com/Harkishen-Singh/Jarvis-personal-assistant/blob/4afba32e7ddf1b16125eaaf9041115762eb7e929/view/components/main.html#L2
https://github.com/Harkishen-Singh/Jarvis-personal-assistant/blob/4afba32e7ddf1b16125eaaf9041115762eb7e929/view/components/main.html#L25

I didn't know the design was changed. I'll remove the background from the body for now.